src/Request/Oembed/GuzzleDispatcher.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Bolt\Request\Oembed;
```

Comment:
I know this is making a request to (O)embed, but I don't think it belongs here. It has nothing to do with our Symfony HTTP Requests. Instead it's a service we are calling. I think `Bolt\(O)embed` would be better.